storage_service/storage_service/settings/production.py

```diff
@@ -4,53 +4,45 @@
 from os import environ
 
 from .base import *  # noqa: F401, F403
-from .base import get_env_variable  # To make he linter happy
-
-# Normally you should not import ANYTHING from Django directly
-# into your settings, but ImproperlyConfigured is an exception.
-from django.core.exceptions import ImproperlyConfigured
-
-
-
-
-def get_env_setting(setting):
-    """ Get the environment setting or return exception """
-    try:
-        return environ[setting]
-    except KeyError:
-        error_msg = "Set the %s env variable" % setting
-        raise ImproperlyConfigured(error_msg)
-
 
 # ######## HOST CONFIGURATION
 # See: https://docs.djangoproject.com/en/1.5/releases/1.5/#allowed-hosts-required-in-production
 ALLOWED_HOSTS = ['*']
 # ######## END HOST CONFIGURATION
 
 # ######## EMAIL CONFIGURATION
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-backend
-EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
 
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-host
-EMAIL_HOST = environ.get('EMAIL_HOST', 'smtp.gmail.com')
+#
+# Which Email backend to use? SMTP is default, others including Amazon SES are
+# also supported.  See https://docs.djangoproject.com/en/dev/ref/settings/ and
+# https://docs.djangoproject.com/en/dev/topics/email/
+#
 
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-host-password
-EMAIL_HOST_PASSWORD = environ.get('EMAIL_HOST_PASSWORD', '')
+EMAIL_BACKEND = environ.get('EMAIL_BACKEND',
+    'django.core.mail.backends.smtp.EmailBackend')
+
+# Amazon SES Backend
+# See https://github.com/azavea/django-amazon-ses
+DJANGO_AMAZON_SES_REGION = environ.get('DJANGO_AMAZON_SES_REGION', 'us-east-1')
 
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-host-user
-EMAIL_HOST_USER = environ.get('EMAIL_HOST_USER', 'your_email@example.com')
 
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-port
+# SMTP Backend
+EMAIL_HOST = environ.get('EMAIL_HOST', 'smtp.gmail.com')
+EMAIL_HOST_PASSWORD = environ.get('EMAIL_HOST_PASSWORD', '')
+EMAIL_HOST_USER = environ.get('EMAIL_HOST_USER', 'your_email@example.com')
 EMAIL_PORT = environ.get('EMAIL_PORT', 587)
+EMAIL_SSL_CERTFILE = environ.get('EMAIL_SSL_CERTFILE')
+EMAIL_SSL_KEYFILE = environ.get('EMAIL_SSL_KEYFILE')
+EMAIL_USE_SSL = environ.get('EMAIL_USE_SSL', 'True') == 'True'
+EMAIL_USE_TLS = environ.get('EMAIL_USE_TLS', 'True') == 'True'
```

Comment:
Should we convert it into boolean? I think Django will only check truthiness and values like "N", "No", or "False" will evaluate as True. I'm not sure though what's the right way to do this in Django. I'd be surprised if there isn't a function in Django to do the conversion easily. Something like [this method in the configparser module](https://github.com/python/cpython/blob/master/Lib/configparser.py#L1156-L1161) which perhaps we can borrow.

Reply:
I don't know about Django builtins but I defined an `is_true` function in settings.base, for detecting boolean env var to switch shibboleth on/off. That could be used here.

Reply:
That's great @helenst, I didn't know (or had forgotten) about that. I agree that is a much better approach and removes the potential for ambiguity here. I'll update the code to use that.